app/forms/openid_connect_authorize_form.rb

```diff
@@ -206,6 +206,7 @@ def extra_analytics_attributes
       scope: scope&.sort&.join(' '),
       acr_values: acr_values&.sort&.join(' '),
       unauthorized_scope: @unauthorized_scope,
+      code_digest: code ? Digest::SHA256.hexdigest(code) : nil,
```

Comment:
Rather than reading `code` as a byproduct of `success_redirect_uri`, could we reference it the same way that `success_redirect_uri` is assigning it?

```suggestion
      code_digest: identity&.session_uuid ? Digest::SHA256.hexdigest(identity&.session_uuid) : nil,
```

Reply:
hmmm I like that idea, maybe we make `code` into a getter method and share that in both places? I gave that a shot in 711bb0079410351dbb8efd8b0e2c5678a72d795b